it/lessons/basics/composition.md

```diff
@@ -219,7 +219,7 @@ Se proviamo a chiamare una macro che non è stata ancora caricata, Elixir sollev
 
 ### `use`
 
-Usa il modulo nel contesto corrente. Questo è particolarmente utile quando un modulo ha bisogno di eseguire qualche configurazione. Chiamando `use` invochiamo la funzione (_hook_) `__using__` all'interno del modulo, fornendo al modulo la possibilità di modificare il contesto esistente:
+Usa il modulo nel contesto corrente. Questo è particolarmente utile quando un modulo ha bisogno di eseguire qualche configurazione. Chiamando `use` invochiamo la funzione (_hook_) `__using__` all'interno del modulo, offrendo la possibilità di modificare il contesto esistente:
```

Comment:
"Questo è particolarmente" => "Questo comportamento è particolarmente"
"eseguire qualche configurazione" sounds weird: can config be executed?